### ISSUE TITLE
look for an alternative to tess2

### ISSUE THREAD

**Issue by arturoc** [openframeworks/openFrameworks#4726]
it's always problematic and doesn't seem to be maintained any more. there's also ways to draw complex shapes without tessellation by using the stencil buffer which should be faster and won't rely on any library the main problem is that it involves several state changes that might be problematic

**Comment by ofZach** [comment]
here's a header only tesselator -- I haven't used it, but seems active:   https://github.com/mapbox/earcut.hpp

**Comment by danoli3** [comment]
This is a great idea if we can rip out old ye tess2 with a better replacement! Earcut looks good!

**Comment by armadillu** [comment]
yep been looking for a faster alternative recently. Haven't tested any, but this looked nice 

https://www.cs.cmu.edu/~quake/triangle.html
